jmix-oidc/oidc-starter/src/main/java/io/jmix/autoconfigure/oidc/OidcAutoConfiguration.java

```diff
@@ -143,7 +143,8 @@ public static class OAuth2ResourceServerConfiguration {
         public SecurityFilterChain securityFilterChain(HttpSecurity http,
                                                        JmixJwtAuthenticationConverter jmixJwtAuthenticationConverter,
                                                        ApplicationEventPublisher applicationEventPublisher) throws Exception {
-            http.oauth2ResourceServer(resourceServer -> {
+            http.cors().and()                
```

Comment:
The `org.springframework.security.config.annotation.web.builders.HttpSecurity#cors()` method is deprecated. Could you please replace it with the `org.springframework.security.config.annotation.web.builders.HttpSecurity#cors(org.springframework.security.config.Customizer>)`?
I guess, It should look like `.cors(Customizer.withDefaults())`.

Reply:
for spring used jmix 1.5.x it is not yet deprecated, but the requested change works for that version also.